fapi-client/src/test/scala/com/gu/facia/api/utils/ResolvedMetaDataTest.scala

```diff
@@ -0,0 +1,134 @@
+      val contentWithVideo = contentWithTags(tagWithId("sillyid"))
+      val resolvedMetaData = ResolvedMetaData.fromContent(contentWithVideo)
+     resolvedMetaData.showByline should be (false)
+     resolvedMetaData.showQuotedHeadline should be (false)
+     resolvedMetaData.imageCutoutReplace should be (false)
+     resolvedMetaData.showMainVideo should be (false)
+     resolvedMetaData.isBoosted should be (false)
+     resolvedMetaData.isBreaking should be (false)
+     resolvedMetaData.imageHide should be (false)
+     resolvedMetaData.imageReplace should be (false)
+     resolvedMetaData.showKickerCustom should be (false)
+     resolvedMetaData.showKickerSection should be (false)
+     resolvedMetaData.showKickerTag should be (false)
+    }
+  }
+
+  "Resolving Metadata using fromTrailMetaData" - {
+
+    "Resolve all to false for empty TrailMetaData" in {
+      val resolvedMetaData = ResolvedMetaData.fromTrailMetaData(emptyTrailMetaData)
+      resolvedMetaData.showByline should be (false)
+      resolvedMetaData.showQuotedHeadline should be (false)
+      resolvedMetaData.imageCutoutReplace should be (false)
+      resolvedMetaData.showMainVideo should be (false)
+      resolvedMetaData.isBoosted should be (false)
+      resolvedMetaData.isBreaking should be (false)
+      resolvedMetaData.imageHide should be (false)
+      resolvedMetaData.imageReplace should be (false)
+      resolvedMetaData.showKickerCustom should be (false)
+      resolvedMetaData.showKickerSection should be (false)
+      resolvedMetaData.showKickerTag should be (false)
+    }
+
+    "Resolve all to true for empty TrailMetaData" in {
+      val resolvedMetaData = ResolvedMetaData.fromTrailMetaData(trailMetaDataWithFieldsSetTrue)
+      resolvedMetaData.showByline should be (true)
+      resolvedMetaData.showQuotedHeadline should be (true)
+      resolvedMetaData.imageCutoutReplace should be (true)
+      resolvedMetaData.showMainVideo should be (true)
+      resolvedMetaData.isBoosted should be (false)
+      resolvedMetaData.isBreaking should be (false)
+      resolvedMetaData.imageHide should be (false)
+      resolvedMetaData.imageReplace should be (false)
+      resolvedMetaData.showKickerCustom should be (false)
+      resolvedMetaData.showKickerSection should be (false)
+      resolvedMetaData.showKickerTag should be (false)
+    }
+  }
+
+  "Resolving Metadata using fromContentAndTrailMetaData" - {
+
+    "should resolve correct for cartoon, comment and video when trailMetaData is not set" in {
+      val resolvedCartoon = ResolvedMetaData.fromContentAndTrailMetaData(contentWithCartoon, emptyTrailMetaData)
+      resolvedCartoon.showByline should be(true)
+
+      val resolvedComment = ResolvedMetaData.fromContentAndTrailMetaData(contentWithComment, emptyTrailMetaData)
+      resolvedComment.showByline should be(true)
+      resolvedComment.showQuotedHeadline should be(true)
+      resolvedComment.imageCutoutReplace should be(true)
```

Comment:
When you're testing multiple properties at once, better to use scalatest's `have` syntax?

    resolvedComment should have (
      `showByline (true),
      `showQuotedHeadline (true),
      `imageCutoutReplace (true)
    )